content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -192,6 +192,9 @@ request headers as follows:
   </tbody>
 </table>
 
+{{< warning >}}
+Please note the validation rules within OpenAPI schemas could be incomplete, and additional validation may occur in the codebase. Therefore, it is always recommended to perform a `kubectl apply --dry-run` for thorough verification.
```

Comment:
```suggestion
Please note the validation rules within OpenAPI schemas are incomplete and additional validation occurs in the code of the kube-apiserver. If precise and complete verification is required,  a `kubectl apply --dry-run` runs all the applicable validation.
```

Reply:
Are they _always_ incomplete? Or just often?

Reply:
I'd say ... often. There are some common issues:

- Enum values for fields are deliberately removed from the API spec, for some reasons;
- You can not rely on the spec to tell if a field is optional or required. A lot of them
  are simply incorrect.
- For a required string field, the API doesn't tell you whether an empty string is fine.
- The ObjectMeta struct is shared by all API kinds, and the `name` constraint differs
  across them. But the API spec is generated from the source comment, so you only get
  a "shared" version, where no kind-specific constraints are mentioned.

Reply:
The above are just some examples. There are more inaccuracies in the spec.